ui/raidboss/data/triggers/tsukuyomi-ex.js

```diff
@@ -7,44 +7,52 @@
       id: 'Tsukuyomi Nightfall Gun',
       regex: / 14:2BBC:Tsukuyomi starts using Nightfall/,
       regexDe: / 14:2BBC:Tsukuyomi starts using Einbruch Der Dunkelheit/,
+	  regexFr: / 14:2BBC:Tsukuyomi starts using Jeune Nuit/,
```

Comment:
This all looks good in general, except you have a bunch of tabs here instead of space, so the formatting looks a bit weird.  I fixed this up for you, but in the future see if you can configure your text editor to use spaces instead!